pycontw/pycontw-frontend
# feat: add/modify pages for young inspirers

## Types of changes


* **New feature**

## Steps to Test This Pull Request

1. Click on "Conference"/"young-inspirers" on navigation bar.
2. Scroll down and see if speech boxes can be seen.
3. Click on a speech box and see if frame pops up with further details.

## Expected behavior

![image](https://user-images.githubusercontent.com/26858727/175073579-9dcf85ea-538f-41f2-8476-bc286af6b23d.png)

## Related Issue

#258
